src/components/AccountModal.vue

```diff
@@ -21,99 +21,45 @@
                 <tbody>
                   <tr v-for="comp in compositions" :key="comp.name">
                     <td class="col"><span>{{ comp.name }}</span></td>
-                    <td class="col-1"><button type="button" v-bind:id="comp.name" class="btn btn-outline-secondary" @click="loadNewCompostion">&#9851;</button></td>
+                    <td class="col-1"><button type="button" class="btn btn-outline-secondary">&#9851;</button></td>
```

Comment:
can you add the @click function and the v-bind back, otherwise, we won't be able to load the compositions